terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,77 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                  = "company-requests" # this must match the stack name the service deploys into
+  name_prefix                 = "${local.stack_name}-${var.environment}"
+  global_prefix               = "global-${var.environment}"
+  service_name                = "psc-discrepancies-api"
+  container_port              = "8080"
+  eric_port                   = "10000"
+  docker_repo                 = "psc-discrepancies.api.ch.gov.uk"
+  kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority   = 31
+  lb_listener_paths           = ["/psc-discrepancy-reports*"]
+  healthcheck_path            = "/psc-discrepancy-reports/healthcheck" #healthcheck path for psc-discrepancies-api
```

Comment:
Listener and healthcheck paths need updating for strike-off, see the routes.yaml file for the paths required